docs/rules/no-return-await.md

```diff
@@ -36,6 +32,15 @@ async function foo() {
         return await bar();
     } catch (error) {}
 }
+
+async function foo() {
+    /** 
+     * Note: Though this is not a lint rule violation, 
+     * this function simply proxies a value and does not 
+     * need to be async  
+     */
+    return bar();
+}
```

Comment:
This isn't necessarily true, it depends on `bar`. `foo` always returns a Promise and also never throws, it doesn't have to be the case with `bar`.

This code could be used to ensure that the call does not throw, or to wrap the returned non-promise value from 'bar'.

Reply:
Ok. I've removed the comment. Seems like we've come full circle here so I guess I'll settle for just having the atypical example moved to the bottom.